Translators/WZDx/tim_translator.py

```diff
@@ -0,0 +1,110 @@
+from datetime import datetime
+import os
+import copy
+import logging
+from pgquery import query_db
+from request_wrapper import get_rsu_request, get_sdw_request
+from tim_generator import generate_tim, get_bearing, get_geometry, get_itis_codes
+
+def update_sat_region_name(request, tim_body):
+    new_tim = copy.deepcopy(tim_body)
+    region_name = new_tim['dataframes'][0]['regions'][0]['name']
+    region_name = region_name.replace(
+        'IDENTIFIER', f"SAT_{request['sdw']['recordId']}")
+    if len(region_name) > 63:
+        region_name = region_name[:60] + '...'
+    new_tim['dataframes'][0]['regions'][0]['name'] = region_name
+    return new_tim
+
+
+def update_rsu_region_name(request, tim_body):
+    new_tim = copy.deepcopy(tim_body)
+    region_name = new_tim['dataframes'][0]['regions'][0]['name']
+    region_name = region_name.replace(
+        'IDENTIFIER', f"RSU_{request['rsus'][0]['rsuTarget']}")
+    if len(region_name) > 63:
+        region_name = region_name[:60] + '...'
+    new_tim['dataframes'][0]['regions'][0]['name'] = region_name
+    return new_tim
+
+def active_tim(feature, tim_body):
```

Comment:
This might have bee the one I was referring to previously, feature doesn't seem to be used in this method, we can remove.

Reply:
Removed!